main.py

```diff
@@ -299,30 +333,106 @@ def redirect_uri():
-                observations=url_for('scouting_observations'))
+                   json=json.dumps(observation, indent=4, sort_keys=True),
+                   observations=url_for('scouting_observations'),
+                   attachments=url_for(
+                       'scouting_observation_attachments',
+                       scouting_observation_id=scouting_observation_id),
+                   home=url_for('home'))
 
 
 @app.route('/scouting-observations', methods=['GET'])
 def scouting_observations():
-    observations = climate.get_scouting_observations(state('access_token'), CLIMATE_API_KEY, 100)
+    observations = climate.get_scouting_observations(state('access_token'),
+                                                     CLIMATE_API_KEY,
+                                                     100)
     body = "<p>No Scouting Observations found!</p>"
     if observations:
-        scouting_observations = render_ul(render_scouting_observation_link(o) for o in observations)
-        body = "<p>Your Climate Scouting Observations:{scouting_observations}</p>".format(scouting_observations=scouting_observations)
+        scouting_observations = render_ul(
+            render_scouting_observation_link(o) for o in observations)
+        body = "<p>Your Climate Scouting Observations:\
+        {scouting_observations}</p>".format(
+            scouting_observations=scouting_observations)
 
-    return  """
+    return """
             <h1>Partner API Demo Site</h1>
             {body}
             <p><a href='{home}'>Return home</a></p>
             """.format(body=body, home=url_for('home'))
+
+
+@app.route('/scouting-observation/<scouting_observation_id>/attachments',
+           methods=['GET'])
+def scouting_observation_attachments(scouting_observation_id):
+    ats = climate.get_scouting_observation_attachments(state('access_token'),
+                                                       CLIMATE_API_KEY,
+                                                       scouting_observation_id)
+
+    body = "<p>No attachments found!</p>"
+    if ats:
+        attachments = render_ul(
+            render_attachment_link(scouting_observation_id, a) for a in ats)
+        body = "<p>Your Climate Scouting Observations attachments:\
+        {attachments}</p>".format(
+            attachments=attachments)
+
+    return """
+            <h1>Partner API Demo Site</h1>
+            {body}
+            <p><a href='{attachments}'>Return to Observation:{soid}</a></p>
+            <p><a href='{home}'>Return home</a></p>
+            """.format(body=body,
+                       home=url_for('home'),
+                       attachments=url_for(
+                           'scouting_observation',
+                           scouting_observation_id=scouting_observation_id),
+                       soid=scouting_observation_id)
+
+
+@app.route(
+    '/scouting-observation/<scouting_observation_id>/attachments/<aid>',
```

Comment:
do you prefer shorthand like `aid` or verbose, like `scouting_observation_id`? for consistency, we should pick one approach and stick with it

Reply:
Updated as attachment_id